### PR TITLE
API Mock Test

### PR DESCRIPTION
**Pull request by JBergen-Dev**
Successfully created API Mock Test and solved merge conflicts that arose from linting in the main branch.

image

### CODE REVIEW

File: api_mock_test.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import unittest 
+from unittest.mock import patch
+import mocks
+import search
+
+class TestingApiCalls(unittest.TestCase):
+    @patch('search.sp.search')
+    def testSearch(self, mockcreate):
```
Comment (KinjalRele) [inline api_mock_test.py:8]: Can you add DocString here?

**Review by KinjalRele — COMMENTED** [review]
Everything else looks good!

File: api_mock_test.py (hunk 1 above)
Comment (JBergen-Dev) [reply api_mock_test.py:8]: Will add this in PR for unit testing

Comment (KinjalRele) [reply api_mock_test.py:8]: cool

**Review by KinjalRele — APPROVED** [review]

**Pull request merged**